source/retryable-writes/tests/README.rst

```diff
@@ -395,6 +395,50 @@ and sharded clusters.
 
    9. Disable the failpoint.
 
+#. Test that drivers return the original error after encountering an error with
+   label `NoWritesPerformed`. This test MUST be implemented by any driver that
+   implements the Command Monitoring specification.
```

Comment:
Noting @ShaneHarvey's concerns about implementing this test below, it made me think back to whether we have existing tests for returning "original errors". There are several other points in the Retryable Writes spec that instruct drivers to do so, but it doesn't appear we have tests (neither spec nor prose) for any of those conditions.